src/sink.cpp

```diff
@@ -117,23 +101,6 @@ static nlohmann::json init_context_info_impl(Log const& log,
     return std::move(j);
 }
 
-void octo::logger::unittests::init_context_info(nlohmann::json& dst,
```

Comment:
those functions are not UT protected, are we sure we can delete them?

Reply:
These are helpers for the UT, not the other way around (see their namespace as well)